packages/render-ui/src/__tests__/parseHtmlHasText.test.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { parseHtmlHasText } from '../parseHtmlHasText';
```

Comment:
`hasText`? or `hasContent` - you're looking for a boolean so typically you have a `is*` or `has*` function name? also the file name I think we use `kebab-case` if that's the case - make it consistent